src/core/Select/Select.tsx

```diff
@@ -270,11 +270,10 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
       <DropdownMenu
         menuItems={menuItems}
         placement='bottom-start'
-        className={menuClassName}
+        className={cx('iui-scroll', menuClassName)}
```

Comment:
Why we needed separate class for this? Wouldnt it make sense that its always scrollable?

Reply:
There must be some reason that previously we didn't even have it in the css.

Maybe Select makes sense to always be scrollable, but not other Menus.

Reply:
which ones not?

Reply:
idk, just a normal menu.

I'm not seeing the problem here. All I did was move `overflow` from inline style into a css class.
![image](https://user-images.githubusercontent.com/9084735/125804085-4b9f3cc8-bcf0-4c28-be3e-ca6b732c42af.png)

We could make it default but that will be a separate css release.